.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+* Title: A short but descriptive summary of the issue, whether it be a bug or enhancement.
+* Attachments: For bugs, attach the agent log and configuration files
+  * /etc/neutron/services/f5/f5-openstack-agent.ini
```

Comment:
Add:
/var/log/neutron/server.log
/etc/neutron/plugins/ml2/ml2_conf.ini

Instead of asking for the agent log or configuration file, I would rather see the output from neutron agent-list and neutron agent-show <agent_uuid>